src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -44,12 +44,14 @@ class HtmlDumper extends CliDumper
         'key' => 'color:#56DB3A',
         'index' => 'color:#1299DA',
         'ellipsis' => 'color:#FF8400',
+        'ns' => 'user-select:none;',
     );
 
     private $displayOptions = array(
         'maxDepth' => 1,
         'maxStringLength' => 160,
         'fileLinkFormat' => null,
+        'notSelectableChars' => array("\n" => true, "\r" => true),
```

Comment:
Not sure about the need to make this configurable: these are the only chars that are "doubly" printed. Let's make things simpler.